API/common/reporter.py

```diff
@@ -77,3 +78,52 @@ def report_final(testresults):
     console.log('  FAIL:    %d' % results['fail'], console.TERMINAL_RED)
     console.log('  TIMEOUT: %d' % results['timeout'], console.TERMINAL_RED)
     console.log('  SKIP:    %d' % results['skip'], console.TERMINAL_YELLOW)
+
+
+def report_coverage(coverage_output):
+    with open(coverage_output) as file:
+        json_data = json.load(file)
```

Comment:
Please don't use built-in names for variables. (There was already a PR (https://github.com/Samsung/js-remote-test/pull/123) about that.)